OpenRA.Game/TextNotificationsManager.cs

```diff
@@ -56,7 +64,7 @@ public static void Debug(string s, params object[] args)
 			AddSystemLine("Debug", string.Format(s, args));
 		}
 
-		static void AddTextNotification(TextNotificationPool pool, string prefix, string text, Color? prefixColor = null, Color? textColor = null)
+		private static void AddTextNotification(TextNotificationPool pool, string prefix, string text, Color? prefixColor = null, Color? textColor = null)
```

Comment:
"Add transient notifications pool " adds a stray `private` here which is removed by a later commit.

Can we please edit this out from the first commit?